snowflake/udfs/udfs/policy_recommendation/policy_recommendation_udf.py

```diff
@@ -18,7 +18,7 @@ def generate_policy_name(info):
 
```

Comment:
I feel like technically, we should bump the version of the policy recommendation function, since we modified all that code. Maybe to v0.1.1?

Reply:
Definitely, thanks for reminding.